app/assets/javascripts/app/views/publisher_view.js

```diff
@@ -103,56 +103,50 @@ app.views.Publisher = Backbone.View.extend({
   },
 
   initSubviews: function() {
-    var form = this.$('.content_creation form');
+    var form = this.$(".content_creation form");
 
-    this.view_services = new app.views.PublisherServices({
-      el:    this.$('#publisher_service_icons'),
-      input: this.el_input,
-      form:  form
-    });
```

Comment:
`view_services` may not be used anywhere else but the `new app.views.PublisherServices(...)` call is important.